ci-operator/step-registry/upi/conf/openstack/upi-conf-openstack-chain.yaml

```diff
@@ -0,0 +1,17 @@
+chain:
+  as: upi-conf-openstack
+  steps:
+  - ref: ipi-conf-openstack
```

Comment:
This step is not available any more. You may want to set:
- ref: openstack-conf-clouds
- ref: openstack-conf-resourcenames

Reply:
done